File: zbus-api/zbus-api-nodejs/examples/consumer.js
var zbus = require("../zbus");
var Message = zbus.Message;
var RemotingClient = zbus.RemotingClient;
var Consumer = zbus.Consumer;


var client = new RemotingClient("127.0.0.1:15555");
client.connect(function(){
    var consumer = new Consumer(client, "MyMQ");
    consumer.recv(function(msg){
        console.log(msg.toString());
    });
});

File: NET/Zbus.Examples/Mq/Consumer.cs

using Zbus.Mq;
using Zbus.Broker;
using Zbus.Net.Http;
using System.Text;
using System;
using System.Security.Cryptography;

namespace Zbus.Examples.Mq
{

   class MyHandler : IMessageHandler
   {
      public void Handle(Message msg, Consumer consumer)
      {
         System.Console.WriteLine(msg);
      }
   }

   class ConsumerTest
   { 
      public static void Main(string[] args)
      {
         IBroker broker = new ZbusBroker(); //using BrokerConfig to change default

         Consumer c = new Consumer(broker, "MyMQ");
         c.ConsumeTimeout = 30000;

         c.OnMessage(new MyHandler());
         c.Start();
         //c.Stop();

         //broker.Dispose();
      }
   }
}
